Mathlib/Analysis/NormedSpace/PiLp.lean

```diff
@@ -91,14 +91,12 @@ variable (p : ℝ≥0∞) (𝕜 𝕜' : Type _) {ι : Type _} (α : ι → Type
 
 /-- Canonical bijection between `PiLp p α` and the original Pi type. We introduce it to be able
 to compare the `L^p` and `L^∞` distances through it. -/
-protected def equiv : PiLp p α ≃ ∀ i : ι, α i :=
-  Equiv.refl _
+abbrev equiv : PiLp p α ≃ ∀ i : ι, α i := WithLp.equiv _ _
```

Comment:
Do we really want to keep this? all elementary lemmas are now called `WithLp.equiv_foo` so it might easier to have `WithLp.equiv` for everything

Reply:
I figured it would be easier to clean things up in a follow-up PR. I think this is quite a lot of work, and I didn't want to block your ProdLp PR while doing it.

Reply:
#6501